x/docs/internal/types/msgs.go

```diff
@@ -122,3 +122,49 @@ func (msg MsgShareDocument) GetSignBytes() []byte {
 func (msg MsgShareDocument) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{msg.Sender}
 }
+
+// ----------------------------------
+// --- DocumentReceipt
+// ----------------------------------
+
+type MsgDocumentReceipt types.DocumentReceipt
```

Comment:
I think we can rename this to something like `MsgSendDocumentReceipt` as it would be more clear.

Reply:
This is still open @bragaz